Automattic/woocommerce-payments
# Update timeline details rendering for multi-currency transactions

Part of #1054

#### Changes proposed in this Pull Request

* Add foreign exchange (FX) details to transaction timeline for multi-currency transactions.

**TODO:**

- [x] Test and fix exchange rate string formatter.

#### Testing instructions

* All checks must be green.

**Feature testing:**

* Requires 534-gh-Automattic/woocommerce-payments-server for testing. Changes from https://github.com/Automattic/woocommerce-payments/pull/1250 may be needed to create transactions from the client.
* Set store currency to any non-USD value.
* Create various non-USD transactions ([test cards](https://stripe.com/docs/testing)): normal, failed, partially refunded (including multiple refunds), fully refunded, disputed without evidence, disputed with winning evidence (submit evidence with `winning_evidence` text in additional details to win), disputed with lost evidence (submit evidence with `losing_evidence` text in additional details to win).
* For each case check transaction timeline view matches designs (pc2DNy-9L-p2).

**Testing without server PR**

* Checkout main server branch.
* Check timeline for each of the transactions above. It should render numbers as before this feature and without crashes.

-------------------
